src/modules/RearSystemCheck.rb

```diff
@@ -94,6 +94,15 @@ def SystemCheckDisk
       supportedfs = [:ext2, :ext3, :ext4, :tmpfs, :swap, :none, :nfs, :nfs4, :btrfs, :xfs]
       unsupported = []
 
+      # Check rear version
+      rear_cmd_ver = "/usr/sbin/rear -V | cut -d' ' -f2";
+      out = SCR.Execute(path(".target.bash_output"), rear_cmd_ver);
+
+      # version >=1.18 supports  vfat partitions
+      if Ops.get_string(out, "stdout", "") >= "1.18"
```

Comment:
this is not how to compare versions.
Try it yourself in ruby
```
jreidinger@localhost:~/prace/yast/skelcd-control-suse-manager-server/package> irb
irb(main):001:0> "1.9" >= "1.17"
=> true
```

So what is needed here is to compare real versions.

Reply:
There are several options:

- [Pkg.CompareVersions](CompareVersions) - this uses the libzypp functionality, this one should be preferred for comparing RPM package version
- Ruby provides native `Gem::Version` class which can be used like
  `if Gem::Version.new(version) >= Gem::Version.new("1.18")`

Fo simple version numbers like `1.18` both should work well.

Reply:
Sorry, my bad, this is not how to compare versions in any language, not just in ruby. Will update the PR.